backend/src/v5/processors/teamspaces/projects/models/commons/tickets.groups.js

```diff
@@ -14,22 +14,39 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+/* eslint-disable no-underscore-dangle */
 
 const { UUIDToString, stringToUUID } = require('../../../../../utils/helper/uuids');
-const { getGroupById, updateGroup } = require('../../../../../models/tickets.groups');
+const { addGroups, getGroupById, updateGroup } = require('../../../../../models/tickets.groups');
+const { getMetadataByQuery, getMetadataByRules } = require('../../../../../models/metadata');
+const { getNodesByIds, getNodesBySharedIds } = require('../../../../../models/scenes');
 const { getFile } = require('../../../../../services/filesManager');
 const { getLatestRevision } = require('../../../../../models/revisions');
-const { getMetadataByRules } = require('../../../../../models/metadata');
-const { getNodesBySharedIds } = require('../../../../../models/scenes');
 
 const TicketGroups = {};
 
+const externalIdNamesToKeys = {
+	ifc_guids: ['IFC GUID', 'Ifc::IfcGUID', 'Element::IfcGUID'],
```

Comment:
I think you're missing IFC Guids from RVT file 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/f81040e4-a0ae-4a6a-9501-b89bfcc6b9a4)
https://staging.dev.3drepo.io/v5/viewer/DevTeamWorkspace/aa721c11-4f87-491c-b18e-002a8505cf9d/72d992ea-026d-4312-8bbc-c80c658e033f